cmd/brew-brewdle.rb

```diff
@@ -18,9 +19,10 @@
 
   brew brewdle                    read Brewfile and install all dependencies
   brew brewdle dump               write all currently installed packages into a Brewfile
+  brew brewdle cleanup            uninstall all Homebrew formulae not listed in Brewfile
 
   Options:
-  --force                        force overwrite existed Brewfile
+  --force                        force overwrite existed Brewfile or cleanup without prompting
```

Comment:
Sorry, I think the `force overwrite existed Brewfile` bit doesn't quite make sense so we could maybe tweak that to the previously suggested string too? Sorry, last thing and then will merge!